build.gradle

```diff
@@ -48,20 +52,24 @@ repositories {
         url = "https://maven.mcmoddev.com/"
     }
     maven {
-        name = "minecraftforge"
-        url = "https://files.minecraftforge.net/maven/"
+        name = "Overmind forge repo mirror"
+        url = "https://gregtech.overminddl1.com/"
     }
     maven {
-        name = "jeknins-thirdparty"
-        url = "http://jenkins.usrv.eu:8081/nexus/content/repositories/thirdparty/"
+        name = "forge"
+        url = "https://files.minecraftforge.net/maven"
```

Comment:
Isn't this https://maven.minecraftforge.net no?

Reply:
Looks like this url is working again, but I'll change anyway.

Reply:
done